Makefile

```diff
@@ -40,4 +40,5 @@ format:
 
 verify:
 	black . --check
+	ruff .
```

Comment:
can you add this to the `format` target also?

Reply:
it does not do any formatting